charts/pega/templates/_pega-k8s-ingress.tpl

```diff
@@ -14,9 +14,15 @@ spec:
   # To access the below service, along with {{ .node.domain }}, traefik http port also has to be provided in the URL.
   - host: {{ .node.service.domain }}
     http:
-      paths:
-      - backend:
-          serviceName: {{ .name }}
+      paths: 
```

Comment:
This seems to only be implemented in the k8s-ingress.  What about the others?

Any HTTPS changes?

Reply:
Currently there is no path set for https in current _pega-k8s-ingress.tpl. 
Please let us know if required we can add entry for https in _pega-k8s-ingress.tpl

Reply:
Discussed with @whitewalkers, we will add in all the ingress files. 
And no need to give entry for https.

Reply:
HTTPS/TLS will be configured on all clouds by default.  Suggest we verify as part of cloud validation.